charts/catalog/values.yaml

```diff
@@ -18,15 +18,7 @@ apiserver:
     # Type of service; valid values are "LoadBalancer" and "NodePort"
     # NodePort is useful if deploying on bare metal or hacking locally on
     # minikube
-    type: NodePort
-    # Further configuration for services of type NodePort
-    nodePort:
-      # Available port in allowable range (e.g. 30000 - 32767 on minikube)
-      # The TLS-enabled endpoint will be exposed here
-      securePort: 30443
-      # Available port in allowable range (e.g. 30000 - 32767 on minikube)
-      # The insecure endpoint, if enabled, will be exposed here
-      insecurePort: 30080
+    type: LoadBalancer
```

Comment:
Ah, I see the charts are reliant on `values.yaml` to supply the defaults. Added back in.